integration-tests/src/test/groovy/com/okta/sdk/tests/it/ApplicationsIT.groovy

```diff
@@ -386,15 +392,12 @@ class ApplicationsIT extends ITSupport {
         registerForCleanup(app)
 
         assertThat(app.status, equalTo(Application.StatusEnum.ACTIVE))
-        slowItDown()
 
         app.deactivate()
         assertThat(client.getApplication(app.getId()).status, equalTo(Application.StatusEnum.INACTIVE))
-        slowItDown()
 
         app.activate()
         assertThat(client.getApplication(app.getId()).status, equalTo(Application.StatusEnum.ACTIVE))
-        slowItDown()
     }
 
 //    @Test
```

Comment:
Any reason this test is commented out? Do we need it?

Reply:
Its actually an issue with the spec, though instead of commenting out, i should have marked it as skipped

(the body is option for this endpoint, but it is the only resource like this, just using the 'required' version is easy

Reply:
Just created https://github.com/okta/openapi/issues/107 (and will add a comment)